generator/html/www.ml

```diff
@@ -436,3 +436,11 @@ let write apis =
     "learn.html";
     "architecture.html";
   ]
+
+let _ =
+  let open Types in
+  let open Files in
+  let apis = [ Plugin.api; Control.api; Data.api; ] in
+  (* Prepend the debug_info argument *)
+  let apis = List.map Types.prepend_dbg apis in
+  write apis
```

Comment:
Would this require any changes in `xapi-storage-script`?

Reply:
Presumably, because I renamed the library to match the package name. We should label this as "don't merge" until the other PRs arrive.